src/main/java/mate/controller/AddCarController.java

```diff
@@ -0,0 +1,65 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.dao.CarDaoImpl;
+import mate.dao.ManufacturerDaoImpl;
+import mate.model.Car;
+import mate.model.Manufacturer;
+import mate.service.CarService;
+import mate.service.CarServiceImpl;
+import mate.service.ManufacturerService;
+import mate.service.ManufacturerServiceImpl;
+
+public class AddCarController extends HttpServlet {
+    private ManufacturerService manufacturerService =
+            new ManufacturerServiceImpl(new ManufacturerDaoImpl());
+    private CarService carService = new CarServiceImpl(new CarDaoImpl());
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        List<Manufacturer> all = manufacturerService.getAll();
+        req.setAttribute("allManufacturer", all);
+        req.getRequestDispatcher("/WEB-INF/views/addCarViews.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Car car = null;
+
+        String model1 = req.getParameter("model_1");
+        String name = req.getParameter("name");
```

Comment:
What is going here? You should have only `model `and `manufacturer_id `parameters - that is all
do not create manufacturer